### PR TITLE
feat(docs): update docs and add new screenshots

### PR DESCRIPTION
**Pull request by OPesicka**

### CODE REVIEW

File: apps/docs/src/pages/creating-flows/start-flow.mdx [hunk 1]
```diff
@@ -1,28 +1,69 @@
...
+Similarly to **Input change**, the **Form submit** option starts a flow when the user submits a specific form. Use a [CSS selector](#how-to-pick-the-right-css-selector) that matches the form element.
+
+## How to pick the right CSS selector
+
+Selecting elements on the page requires the right CSS selector to avoid conflicts with other elements. Here are some tips:
+
+- **Use unique classes or IDs**: If possible, use unique classes or IDs for the target element to ensure the selector matches only the desired element.
+- **Use the most specific selector possible**: If unique classes or IDs aren't available, use the most specific selector possible to reduce conflicts.
+- **Use browser developer tools**: Use browser developer tools to find the right selector. This [guide](https://www.scrapingbee.com/tutorials/how-to-extract-css-selectors-using-chrome/) can help you find the right CSS selector.
+- **Test the selector**: After selecting a selector, test it in the browser's console to ensure it matches the correct element. Use `document.querySelector` for testing.
```
Comment (VojtechVidra) [inline apps/docs/src/pages/creating-flows/start-flow.mdx:61]: I would change to something like:
Use `document.querySelectorAll(".my-selector")` for testing and make sure there's only one result.

File: apps/docs/src/pages/creating-flows/start-flow.mdx [hunk 2]
```diff
@@ -1,28 +1,69 @@
...
+
+### Input change
+
+The **Input change** option starts a flow when the user changes the value of a specific input, like a text input in a form. Use a [CSS selector](#how-to-pick-the-right-css-selector) that matches the input element.
+
+### Form submit
+
+Similarly to **Input change**, the **Form submit** option starts a flow when the user submits a specific form. Use a [CSS selector](#how-to-pick-the-right-css-selector) that matches the form element.
+
+## How to pick the right CSS selector
```
Comment (VojtechVidra) [inline apps/docs/src/pages/creating-flows/start-flow.mdx:54]: I would consider creating a dedicated page for this section. What do you think?

File: apps/docs/src/pages/creating-flows/frequency.mdx [hunk 3]
```diff
@@ -1,17 +1,25 @@
...
-and are not shown to the user too often.
+- **Every time**: The flow will be shown every time the user meets all flow conditions. We recommend using this option only for flows that are not intrusive
+  and are not shown to the user too often.
+
+## Reset user progress
+
+When the flow is set to show only once, you can reset the progress for a selected user. This can be useful when you are testing the flow and want to see it again, or when you want to show the flow to the user again after some time.
+
+<SectionLink href="/project-setup/reset-user-progress">
+  Learn more about resetting flow eligibility →
```
Comment (VojtechVidra) [inline apps/docs/src/pages/creating-flows/frequency.mdx:24]:
```suggestion
  Learn more about resetting flow progress →
```

File: apps/docs/src/pages/creating-flows/start-flow.mdx (hunk 2 above)
Comment (OPesicka) [reply apps/docs/src/pages/creating-flows/start-flow.mdx:54]: Good idea. Added a todo for later

**Review by VojtechVidra — APPROVED** [review]

**Pull request merged**